### DISCUSSION TITLE
Error Boundaries

### DISCUSSION THREAD

**Discussion by ehubbell** [jaredpalmer/razzle#1328] [Q&A] [answered]
## ❓Question

Curious if there is a recommended pattern for error boundaries / error handling with Razzle? According to the [React docs](https://reactjs.org/docs/error-boundaries.html), Error Boundaries don't work with SSR and I've come across this with our application. Any input would be appreciated!

**Comment by fivethreeo** [comment, marked as answer]
I have been pondering this as well.

Closest info about this I have seen is:
https://dev.to/spukas/error-boundaries-with-react-23ec

**Reply by fivethreeo** [reply]
This is a unsolved React issue, server-components looks promising.

https://github.com/jaredpalmer/razzle/tree/canary/examples/with-react-server-components
